File: VB/HSRer.vb
Module HSRer
    Sub Main(args As String())

        Console.WriteLine("Press anything to derail stars...")
        Console.ReadKey(False)

        Dim Processes = Process.GetProcessesByName("StarRail")

        If Processes.Length > 0 Then
            For Each GameProcess In Processes
                GameProcess.Kill()
            Next

            Console.WriteLine("Derailed all the stars.")
        Else
            Console.WriteLine("No stars to derail...")
        End If

        Console.ReadKey(False)

    End Sub
End Module

File: C#/HSRer.cs
using System.Diagnostics;

internal class HSRer
{
    private static void Main(string[] args)
    {
        Console.Title = "HSRer";

        Console.WriteLine("Press anything to derail stars...");
        Console.ReadKey(true);

        var ProcessesByName = Process.GetProcessesByName("StarRail");
        if (ProcessesByName.Length != 0)
        {
            foreach (var Process in ProcessesByName)
                if (Process != null)
                    Process.Kill();

            Console.WriteLine("Derailed all the stars.");
        }
        else
            Console.WriteLine("No stars to derail...");

        Console.ReadKey(true);
    }
}
